woocommerce/woocommerce
# Add Pinterest Extension to Onboarding Wizard

### All Submissions:






### Changes proposed in this Pull Request:



Closes (partially) https://github.com/woocommerce/woocommerce/issues/34302.

Adds Pinterest for WooCommerce to the Onboarding Wizard.






### How to test the changes in this Pull Request:



With a new site,

1. Skip the Onboarding Wizard.
2. Disable "Display suggestions within WooCommerce " in Settings > Advanced > WooCommerce.com.
3. Navigate to the Free Features section in the Onboarding Wizard.
4. See that Pinterest is listed.
5. Select Pinterest and complete the Onboarding Wizard.
6. On the WordPress Plugins page, confirm that Pinterest for WooCommerce plugin is installed.

![image](https://user-images.githubusercontent.com/9312929/207806631-b8d42ca3-16a6-4ac8-8c0a-6dbe09df44ab.png)




### Other information:




### FOR PR REVIEWER ONLY:
